src/python/pants/backend/python/goals/export.py

```diff
@@ -181,17 +181,22 @@ async def do_export(
             digest=merged_digest,
             post_processing_cmds=[
                 PostProcessingCommand(
-                    [
-                        requirements_pex.python.path,
+                    complete_pex_env.create_argv(
                         os.path.join(pex_pex_dest, pex_pex.exe),
-                        os.path.join("{digest_root}", requirements_pex.name),
-                        "venv",
-                        "--pip",
-                        "--collisions-ok",
-                        "--remove=all",
```

Comment:
@benjyw was this the cause of https://github.com/pantsbuild/pants/issues/17221? It seems likely.